### ISSUE TITLE
[Bug]: SELECT * FROM mo_catalog.mo_locks error: can not get view statement

### ISSUE THREAD

**Issue by aylei** [matrixorigin/matrixone#15441]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

1.1-dev

### Commit ID

1519f8e5c

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

```
mysql> SELECT * FROM mo_catalog.mo_locks limit 1;                                                                                                                                                               ERROR 1064 (HY000): SQL parser error: can not get view statement
mysql> select git_version();
+---------------+
| git_version() |
+---------------+
| 1519f8e5c     |
+---------------+
1 row in set (0.08 sec)
```

### Expected Behavior

SQL succeed

### Steps to Reproduce

```Markdown
mysql -h freetier-02.cn-hangzhou.cluster.cn-qa.matrixone.tech -u'dump?role=__MO_OB__' -P6001 -p

Then execute the SQL
```

**Comment by daviszhen** [comment]
未投入

**Comment by qingxinhome** [comment]
待处理

**Comment by qingxinhome** [comment]
The issue has been resolved and submitted to 1.1 dev and main for verification testing @aylei @sukki37 @heni02

**Comment by heni02** [comment]
confirm,closed
main commit:341c2b6e358d776186e20a723bf96849e409ccff
sys account：
<img width="1417" alt="企业微信截图_c1dce9dc-ea7e-4cd0-88ab-fbddc1c53695" src="https://github.com/matrixorigin/matrixone/assets/113406637/9cfa0548-f086-4298-9b44-1c075776e916">

non sys account:
![image](https://github.com/matrixorigin/matrixone/assets/113406637/43efad56-c4ec-491d-9e4d-3a9fbd61775d)

1.1-dev commit:09d8b10620aafcc1af33510a08608eeb0c3c0d5f
sys account:
<img width="1876" alt="企业微信截图_2a759000-0ecf-4b4b-b2a1-b3a8e0825423" src="https://github.com/matrixorigin/matrixone/assets/113406637/cead0f98-4c37-4221-9fc5-33972a6ff8e1">

non sys account:
<img width="1874" alt="企业微信截图_daabad9a-2da4-4741-856f-4a5501cbaa81" src="https://github.com/matrixorigin/matrixone/assets/113406637/2db604c8-ece0-4477-a1d1-cf54dc886766">
